README.md

```diff
@@ -2,13 +2,104 @@
 
 # Project Details
 
+In this sprint, we are building a front-end application that allows users to load, view and search CSV files. We use HTML and CSS to set up a basic command-prompt interface, and TypeScript with React to organize components and process events in that interface. We used mocked data in place of CSV files to mock the functionality of our load, view, and search functions.
+
 # Design Choices
 
+Our project splits the program into the following components, each handling a different function of the program. App.tsx is the top level component that handles the Login button. REPL.tsx sets up the REPL interface and allows the REPLInput and REPLHistory components to share information passed in by the user. Finally, the Commands component contains the load, view, and search commands as well as the mocked datasource.
```

Comment:
The current project organization could perhaps be compartmentalized into packages or folders based on functionality to make the code easier to read